### ISSUE TITLE
Check for Peregrine friendly filenames

### ISSUE THREAD

**Issue by richelbilderbeek** [richelbilderbeek/raket#43]
There are Peregrine-unfriendly filenames in the `raket` parameters, e.g. (from log below) `/local/tmp/RtmpcVLdF8/file1cc08453580ed.log`. Check for these.

From a Peregrine run:

```
Rscript -e raket::run_raket_from_file("./data/10/parameters.RDa")
Registered S3 method overwritten by 'geiger':
  method            from
  unique.multiPhylo ape 
Error in value[[3L]](cond) : Could not estimate the marginal likelihood. 
Error message: File 'output_log_filename' not found. Could not find file with path '/local/tmp/RtmpcVLdF8/file1cc08453580ed.log'
site_model$name: JC69
clock_model$name: strict
tree_prior$name: birth_death
epsilon: 1e-12
rng_seed: NA
verbose: FALSE
beast2_working_dir: /home/p230198/.cache/file1cae877f27cff/beast2_tmp_folder1cae84973ab1d
beast2_bin_path: /home/p230198/.local/share/beast/bin/beast
Calls: <Anonymous> ... tryCatch -> tryCatchList -> tryCatchOne -> <Anonymous>
In addition: Warning messages:
1: In file.rename(from = actual_log_filename, to = output_log_filename) :
  cannot rename file '/home/p230198/.cache/file1cae877f27cff/beast2_tmp_folder1cae84973ab1d/pbd_evidence.log' to '/local/tmp/RtmpcVLdF8/file1cc08453580ed.log', reason 'Invalid cross-device link'
2: In file.rename(from = actual_trees_filename, to = to) :
  cannot rename file '/home/p230198/.cache/file1cae877f27cff/beast2_tmp_folder1cae84973ab1d/pbd_evidence.trees' to '/local/tmp/RtmpcVLdF8/file1cc0815a59692.trees', reason 'Invalid cross-device link'
Execution halted
```

**Comment by richelbilderbeek** [comment]
Maybe the error is upstream, see [pirouette Issue 9](https://github.com/richelbilderbeek/peregrine/issues/9)

**Comment by richelbilderbeek** [comment]
Fixed with mcbette v1.6
